### DISCUSSION TITLE
[vue] Pass additional props to <svg> element

### DISCUSSION THREAD

**Discussion by PssbleTrngle** [tailwindlabs/heroicons#887] [Help]
It seems like it is not possible to pass additional props to the `<svg>` element, expect for the `class`

```vue
<MapPinIcon class="h-2 w-2" data-test="something" />
```

I know the icons are meant to be resized using css height/width, but I am trying to insert icons nested into another svg element and it seems like I need to pass them the actual height/width prop

**Comment by mitar** [comment]
I am in exactly the same problem. Using Vue components inside another SVG seems problematic because attributes are not passed through.

**Reply by mitar** [reply]
OK, it looks like one can wrap icon in yet another `<svg>` element to control the size:

```
      <svg x="0" y="0" width="22" height="22">
        <StarIcon style="color: #f29727" />
      </svg>
```
